web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -62,4 +63,55 @@ describe("<SidebarCardsList />", () => {
     fireEvent.click(screen.getByTestId("hidden-opportunity-header"));
     expect(mockAnalytics.event.for_you_card_unhide_button.click.unhide_cards).toHaveBeenCalledTimes(1);
   });
+
+  describe("empty state messaging", () => {
+    it("displays when the For You section is empty", () => {
+      render(
+        <SidebarCardsList
+          topCards={[]}
+          bottomCards={[]}
+          fundings={[]}
+          hiddenFundings={[]}
+          certifications={[]}
+          hiddenCertifications={[]}
+          displayFundings={false}
+          displayCertifications={false}
+        />
+      );
+
+      expect(screen.getByTestId("empty-for-you-message")).toBeInTheDocument();
+      expect(screen.getByAltText("For You Section")).toBeInTheDocument();
+    });
+
+    it("does not display when the For You section has cards", () => {
+      const topCards = [
+        generateSidebarCardContent({
+          id: "some-fake-top-card",
+          section: "above-opportunities",
+        }),
+      ];
+
+      const bottomCards = [
+        generateSidebarCardContent({
+          id: "some-fake-bottom-card",
+          section: "below-opportunities",
+        }),
+      ];
+      render(
+        <SidebarCardsList
+          topCards={topCards}
+          bottomCards={bottomCards}
+          fundings={[]}
+          hiddenFundings={[]}
+          certifications={[]}
```

Comment:
[Boulder] This ticket mentions this the UI is also dependent on funding and certification cards. I don't see the logic for that piece.

Reply:
I think this is covered in the `SidebarCardsContainer.test.tsx` under `displays empty state when all opportunities are hidden because the empty state logic for fundings and certifications already existed, but I think the test can be updated to check that the new image is being displayed.

Reply:
@FarazA22: My understanding is that Design just wanted us to change the image. The logic already exists in our code and shouldn't change. I sent you the Slack convo.